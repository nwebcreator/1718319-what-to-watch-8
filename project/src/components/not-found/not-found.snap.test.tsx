import { render } from '@testing-library/react';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import NotFound from './not-found';

describe('Component: NotFound', () => {
  it('should render correctly', () => {
    const history = createMemoryHistory();
    const { container } = render(
      <Router history={history}>
        <NotFound />
      </Router>,
    );
    expect(container).toMatchSnapshot();
  });
});
