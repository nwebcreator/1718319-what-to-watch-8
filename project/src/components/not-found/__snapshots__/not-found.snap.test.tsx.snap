// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: NotFound should render correctly 1`] = `
<div>
  <a
    class="link-return-url"
    href="/"
  >
    Return to main
  </a>
  <div
    class="flex-center position-r full-height"
  >
    <div
      class="code"
    >
      404
    </div>
    <div
      class="message"
      style="padding: 10px;"
    >
      NOT FOUND
    </div>
  </div>
</div>
`;
